TestAssets/TestProjects/PJLibWithMultipleFrameworks/project.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
It actually doesn't work because we can't build without all those desktop frameworks. So adding the test project back in.